apps/site/components/OverviewSection/overview-section.module.css

```diff
@@ -2,6 +2,7 @@
   display: flex;
   column-gap: 16px;
   align-items: center;
+  justify-content: center;
```

Comment:
I kind prefer when it is aligned to the left, especially when it's just one item 🤔 

<img width="1170" alt="image" src="https://github.com/vtex/faststore/assets/3356699/99f61458-dfb0-4153-9b7d-2d0419399739">